conda/recipes/cudf/meta.yaml

```diff
@@ -38,7 +38,7 @@ requirements:
     - rmm {{ minor_version }}.*
     - nvstrings {{ minor_version }}.*
     - dlpack
-    - fsspec>=0.3.3,<0.7.0a0
```

Comment:
Two things:
1) Since we know a version may have breaking changes, we should bound at least to `<8.0.0a0`
2) Any reason the lower bound is also not increased now? This is a huge range of versions.

Reply:
1. Sure.
2. Will bump the lower bound to `0.6.0` then?
cc: @raydouglass  @kkraus14 
https://github.com/dask/dask/blob/38b9734876d2c18120ca4d0846214cc9b33e1414/setup.py#L14

Reply:
No other library than dask / cudf uses fsspec as far as I know since it's for filesystem interfacing.

I'm okay with bumping the lower bound, but I'd prefer not to restrict the upper bound. People routine struggle with conda solves and restricting things unnecessarily is one of those reasons. The failures here were purely related to unit test code so I think in general we should be fine.

Reply:
Increased only the lower bound for `fsspec`